File: api/api/Models/Feature.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace api.Models
{
    public class Feature
    {
        [Key]
        public int FeatureID { get; set; }

        public string FeatureName { get; set; }

        public virtual List<CategoryFeature> CategoryFeatures { get; set; }

        public virtual List<ProductFeatureValue> ProductFeatureValues { get; set; }

    }
}

File: myAngular/src/app/Project/Models/feature.ts
import { CategoryFeature } from './category-feature';
import { ProductFeatureValue } from './product-feature-value';

export class Feature {
    constructor(
        public FeatureID: number, 
        public FeatureName: string,
        public CategoryFeatures? : CategoryFeature[],
        public ProductFeatureValues? : ProductFeatureValue[]
       
        ) { }
}
